### DISCUSSION TITLE
Editing tags?

### DISCUSSION THREAD

**Discussion by rellimneb** [rern/rAudio#1091] [General]
Is there any way to edit the mp3 tags directly in raudio?

**Comment by EarsofTin** [comment]
One of the options when you click left click on an individual track in library is "Tag Editor". Works for individual tracks and global album edits.

**Reply by rellimneb** [reply]
with left clic i get : add, play next, replace, add similar, add to a playlist and browse directory but no tag editor.  maybe we have different versions if raudio?  using the latest version

**Comment by EarsofTin** [comment]
Using 20221014 and accessing files through the NAS icon on the library tab. I don't get the tag editor when I  access through album, album artist, etc.  I have to click on the "album" (actually a file directory name) to display individual tracks. Then I can edit the album or track by clicking on the album artwork or the individual track.  Edited my prior post to show that album editing works is also possible.

![Screenshot_20221112-152720](https://user-images.githubusercontent.com/76259761/201494163-eafcb957-a776-4adf-8e8d-f1dcacac46c4.jpg)

**Reply by rellimneb** [reply]
ah ok.  I don't have files on a nas, only on an hdd attached to the pi so i can't access this functionality.

**Comment by EarsofTin** [comment]
Try the USB library icon. I suspect you can't edit metadata while accessing music via metadata tags but can if accessing directories via usb or nas.  I will try myself at earliest opportunity.

**Comment by rern** [comment]
It's applicable only in file mode - SD, NAS or USB.
